src/components/ControlBar/ViewControlBar/DashboardsBar.js

```diff
@@ -0,0 +1,133 @@
+import React, { useState, useRef, useEffect, createRef } from 'react'
+import { connect } from 'react-redux'
+import cx from 'classnames'
+import PropTypes from 'prop-types'
+
+import Content from './Content'
+import ShowMoreButton from './ShowMoreButton'
+import DragHandle from './DragHandle'
+import { getRowsFromHeight } from './controlBarDimensions'
+import { sGetControlBarUserRows } from '../../../reducers/controlBar'
+import { acSetControlBarUserRows } from '../../../actions/controlBar'
+import { apiPostControlBarRows } from '../../../api/controlBar'
+
+import classes from './styles/DashboardsBar.module.css'
+
+export const MIN_ROW_COUNT = 1
+export const MAX_ROW_COUNT = 10
+const rowClassMap = {
```

Comment:
As long as a row only needs a `height` prop, I'd argue that these number named classes could be simplified by having a `rowHeightMap` containing objects like `{ 1: 71 }`, which could then be used directly. But maybe that's a bad way to mix css and js?

Reply:
solved using css vars and removing the rowHeightMap